frontend/src/routes/_dashboard/project_/$project_slug.services.docker.$service_slug.tsx

```diff
@@ -0,0 +1,471 @@
+import { Popover } from "@radix-ui/react-popover";
+import { Link, createFileRoute } from "@tanstack/react-router";
+import { addDays, format } from "date-fns";
+import {
+  CalendarIcon,
+  Container,
+  EllipsisVertical,
+  KeyRound,
+  Loader,
+  Rocket,
+  Settings,
+  Timer,
+  TriangleAlert
+} from "lucide-react";
+import * as React from "react";
+import type { DateRange } from "react-day-picker";
+import { withAuthRedirect } from "~/components/helper/auth-redirect";
+import { MetaTitle } from "~/components/meta-title";
+import { StatusBadge } from "~/components/status-badge";
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  BreadcrumbList,
+  BreadcrumbPage,
+  BreadcrumbSeparator
+} from "~/components/ui/breadcrumb";
+import { Button } from "~/components/ui/button";
+import { Calendar } from "~/components/ui/calendar";
+
+import { MultiSelect } from "~/components/ui/mutiple-select";
+import { PopoverContent, PopoverTrigger } from "~/components/ui/popover";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "~/components/ui/tabs";
+import { cn } from "~/lib/utils";
+import { capitalizeText, mergeTimeAgoFormatterAndFormattedDate } from "~/utils";
+
+const statuses = [
+  { value: "QUEUED", label: "QUEUED", color: "gray" },
+  { value: "CANCELLED", label: "CANCELLED", color: "gray" },
+  { value: "FAILED", label: "FAILED", color: "red" },
+  { value: "PREPARING", label: "PREPARING", color: "blue" },
+  { value: "HEALTHY", label: "HEALTHY", color: "green" },
+  { value: "UNHEALTHY", label: "UNHEALTHY", color: "red" },
+  { value: "STARTING", label: "STARTING", color: "blue" },
+  { value: "RESTARTING", label: "RESTARTING", color: "blue" },
+  { value: "REMOVED", label: "REMOVED", color: "gray" },
+  { value: "SLEEPING", label: "SLEEPING", color: "orange" }
+];
+
+export const Route = createFileRoute(
+  "/_dashboard/project/$project_slug/services/docker/$service_slug"
+)({
+  component: withAuthRedirect(ServiceDetails)
+});
+
+function ServiceDetails() {
+  const { project_slug, service_slug } = Route.useParams();
+  const baseUrl = `/project/${project_slug}/services/docker/${service_slug}`;
+  const [date, setDate] = React.useState<DateRange | undefined>({
+    from: new Date(2022, 0, 20),
+    to: addDays(new Date(2022, 0, 20), 20)
+  });
+
+  const [selectedFrameworks, setSelectedFrameworks] = React.useState<string[]>([
+    "react",
+    "angular"
+  ]);
```

Comment:
Rename this variable please, also the default value should be `statuses.map(status => status.value)`.